packages/redux/src/subscriptions/reducer/userSubscriptions.ts

```diff
@@ -286,8 +275,11 @@ const userSubscriptionsReducer: Reducer<UserSubscriptionsState> = (
   action,
 ) => {
   switch (action.type) {
-    case actionTypes.RESET_SUBSCRIPTIONS:
+    case actionTypes.RESET_USER_SUBSCRIPTIONS:
     case authenticationActionTypes.LOGOUT_SUCCESS:
+    case authenticationActionTypes.LOGIN_SUCCESS:
```

Comment:
Question: Is it necessary to have login success, register success and fetch user success actions here as well? I am questioning because the other reducers only listen for the logout_success action type when it needs to clear user data from the store.

Reply:
Never mind, there can be data fetched from guest as well so it might be necessary to listen for those actions as well. I will review reducers that are only looking for the logout success action to check if they might suffer from the same problem.